tests/providers/snowflake/hooks/test_snowflake.py

```diff
@@ -342,6 +344,24 @@ def test_get_conn_params_should_support_private_auth_with_unencrypted_key(
         ), pytest.raises(TypeError, match="Password was given but private key is not encrypted."):
             SnowflakeHook(snowflake_conn_id='test_conn')._get_conn_params()
 
+    def test_get_conn_params_should_fail_on_invalid_key(self):
+        connection_kwargs = {
+            **BASE_CONNECTION_KWARGS,
+            'password': None,
+            'extra': {
+                'database': 'db',
+                'account': 'airflow',
+                'warehouse': 'af_wh',
+                'region': 'af_region',
+                'role': 'af_role',
+                'private_key_file': "/dev/urandom",
+            },
+        }
+        with unittest.mock.patch.dict(
```

Comment:
The `unittest` here is the root cause of the failed checks; delete it to make the checks pass.
```suggestion
        with mock.patch.dict(
```